src/knoop/Node/test/get.test.cpp

```diff
@@ -37,4 +36,12 @@ SCENARIO( "leaf node value extraction" ){
     REQUIRE_THROWS(getInt(sNode));
     REQUIRE_THROWS(getInt(csNode));
   }
+
+  SECTION("a node with <one type> has a handy non-template get()"){
```

Comment:
Not having to specify a template parameter is great. What happens here:

```c++
auto mNode = Node<int, std::string>{"abc"};

auto s = mNode.get();
```
What is the type of `s`? It seems like it is always `int` because that was the first parameter specified.

Reply:
Yes. So, you'd use get() only if you want the int, which in this case you would *not*. But you can still use the get<int> and get<std::string> notation if you want. See below for more details.